soroban-test-wasms/wasm-workspace/Cargo.toml

```diff
@@ -0,0 +1,64 @@
+# This file defines a workspace that is _periodically_ recompiled as part of
```

Comment:
I don't have a concern about this, but I'm asking to learn:

What's the motivation for having these live in a separate workspace?

The SDK has its test vectors in the same workspace, even the test vectors that SDK tests rely on for .wasm files. In the SDK to make sure .wasm files always exist for tests, we just build them for wasm32 before running tests. For example: https://github.com/stellar/rs-soroban-sdk/blob/aa61c62a01cd7a8ba4297889945a7ddd68b46aaf/Makefile#L12-L18

Reply:
A few reasons, each of which may be surmountable -- patches welcome!

  - I wanted them to inherit wasm release-build profile settings, and the outer workspace has different settings
  - I wanted it to be a separate build step (not triggered by an outer-workspace cargo build) and commit the binaries as versioned inputs, so the outer `soroban-test-wasms` crate could be usable from other crates & repos (especially: core) via cargo, without having to run cross-repo make / rebuild them on the fly
  - I wasn't sure what would happen if I patched `soroban-env-common` to point to local _within_ the workspace defining it as local (and I need to patch in order to include the SDK as a dev-dep of the WASMs). Figured keeping the two resolution runs separate would make it easier to be sure about the meaning of things.

I'm happy to iterate if you think there's a cleaner way / think none of these are real issues.

Reply:
I don't feel strongly towards one way or the other. I don't know what the short comings of how I've set it up in the SDK yet. I suspect there are some, maybe lots 😬. Let's see how they play out.